PocketKit/Sources/PocketKit/MyList/SortFilterMenu/View/SortMenuViewController.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+import Lottie
+import Combine
+import Textile
+import SharedPocketKit
+
+class SortMenuViewController: UIViewController {
+
+    private let tableView = UITableView()
+    let viewModel: SortMenuViewModel
+    private var subscriptions: [AnyCancellable] = []
+
+    private lazy var tableViewDataSource: UITableViewDiffableDataSource<SortSection, SortRow> = {
+        let dataSource = UITableViewDiffableDataSource<SortSection, SortRow>(tableView: tableView) { [weak self] tableView, indexPath, itemIdentifier in
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: "SortMenuViewCell") as? SortMenuViewCell else {
+                return UITableViewCell()
+            }
+            cell.model = self?.viewModel.cellViewModel(for: itemIdentifier)
+            return cell
+        }
+        return dataSource
+    }()
+    
+    init(viewModel: SortMenuViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+        
+        tableView.accessibilityIdentifier = "sort-menu"
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        view = tableView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTableView()
+        setupBindings()
+    }
+}
+
+// MARK: View Setup & Configuration
+extension SortMenuViewController {
+    func configureTableView() {
+        tableView.delegate = self
+        tableView.separatorStyle = .none
+        tableView.layer.cornerRadius = 28.0
+        tableView.estimatedRowHeight = 100.0
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.register(SortMenuViewCell.self,
```

Comment:
Style nit: When chopping down method calls, place a newline after the opening paren and place the closing paren on its own line.
e.g:

```swift
tableView.register(
    SortMenuViewCell.self,
    forCellReuseIdentifier: "SortMenuViewCell"
)
```

PS: we are working on adding SwiftLint to help guide folks to using consistent style in #228